netbox/dcim/views.py

```diff
@@ -912,16 +889,16 @@ class DeviceTypeListView(generic.ObjectListView):
 
 
 @register_model_view(DeviceType)
-class DeviceTypeView(generic.ObjectView):
+class DeviceTypeView(GetRelatedModelsMixin, generic.ObjectView):
     queryset = DeviceType.objects.all()
 
     def get_extra_context(self, request, instance):
-        related_models = (
-            (Device.objects.restrict(request.user).filter(device_type=instance), 'device_type_id'),
-        )
-
         return {
-            'related_models': related_models,
+            'related_models': self.get_related_models(request, instance, [
```

Comment:
Including the keyword improves clarity.
```suggestion
            'related_models': self.get_related_models(request, instance, omit=[
```